### PR TITLE
Add custom metric section for Ruby

### PR DESCRIPTION
**Pull request by anmarchenko**
[Jira issue](https://datadoghq.atlassian.net/browse/CIAPP-7001)

### What does this PR do?

Adds a section for custom metrics on Ruby, link to our custom metrics guide, and Ruby tab to our custom metrics guide

### Motivation

Parity between Ruby documentation and docs for other languages

### Additional Notes

---

### Reviewer checklist

- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/ruby.md [hunk 1]
```diff
@@ -397,6 +397,21 @@ Datadog::Tracing.active_span&.set_tag('test_owner', 'my_team')
 
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][5] section of the Ruby custom instrumentation documentation.
 
+### Adding custom metrics to tests
+
+Just like tags, you can add custom metrics to your tests by using the current active span:
```
Comment (estherk15) [inline content/en/continuous_integration/tests/ruby.md:402]:
```suggestion
Like tags, you can add custom metrics to your tests by using the current active span:
```

File: content/en/continuous_integration/tests/ruby.md [hunk 2]
```diff
@@ -397,6 +397,21 @@ Datadog::Tracing.active_span&.set_tag('test_owner', 'my_team')
...
+```ruby
+require 'ddtrace'
+
+# inside your test
+Datadog::Tracing.active_span&.set_tag('memory_allocations', 16)
+# test continues normally
+# ...
+```
+
+Read more about custom metrics in the [Add Custom Metrics Guide][7].
```
Comment (estherk15) [inline content/en/continuous_integration/tests/ruby.md:413]:
```suggestion
For more information on custom metrics, see the [Add Custom Metrics Guide][7].
```

**Review by estherk15 — APPROVED** [review]
Looks good, thanks for updating!

**Review by juan-fernandez — APPROVED** [review]
👏

**Pull request merged**